### ISSUE TITLE
Spikes too weak 

### ISSUE THREAD

**Issue by qqii** [CosyClub/ProjectTempo#182]
Spikes could be easily bypassed

**Comment by AnthonyWharton** [comment]
Agreed, Spikes should be made more powerful, but this is also an issue of level design.

**Comment by qqii** [comment]
That's not the issue, the issues is that with enough combo you can run through any amount of spikes without dying.

**Comment by AnthonyWharton** [comment]
I didn't think as combo increased healing increased? I thought it was an on/off effect.

Also I know, I said that it's a combination of two issues
